source/models/table.py

```diff
@@ -34,11 +37,32 @@ def get_path(self):
     def get_name(self):
         return self.__table_name
 
-    def set(self):
-        pass
+    def set(self, row):
```

Comment:
Move Validate in row constructor

Reply:
sorry my mistake, I did so but forget to remove it from here